src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripDropDownMenu.ToolStripDropDownMenuAccessibleObjectTests.cs

```diff
@@ -0,0 +1,247 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using static Interop.UiaCore;
+
+namespace System.Windows.Forms.Tests
+{
+    public class ToolStripDropDownMenuAccessibleObjectTests : IClassFixture<ThreadExceptionFixture>
+    {
+        public static IEnumerable<object[]> ToolStripDropDownMenu_FragmentNavigate_WithoutItem_TestData()
+        {
+            IEnumerable<Type> types = ReflectionHelper.GetPublicNotAbstractClasses<ToolStripDropDownItem>().Select(type => type);
+            foreach (Type itemType in types)
+            {
+                foreach (bool createControl in new[] { true, false })
+                {
+                    yield return new object[] { itemType, createControl };
+                }
+            }
+        }
+
+        [WinFormsTheory]
+        [MemberData(nameof(ToolStripDropDownMenu_FragmentNavigate_WithoutItem_TestData))]
+        public void ToolStripDropDownMenu_FragmentNavigate_ReturnExpected_WithoutItem(Type itemType, bool createControl)
```

Comment:
Please add `AccessibleObject` for all test names, because you test not control but its AO
```suggestion
        public void ToolStripDropDownMenuAccessibleObject_FragmentNavigate_ReturnExpected_WithoutItem(Type itemType, bool createControl)
```